CMakeLists.txt

```diff
@@ -171,6 +171,25 @@ INSTALL(DIRECTORY ${CMAKE_BINARY_DIR}/lib DESTINATION ${CMAKE_INSTALL_PREFIX} PA
 
 INSTALL(CODE "execute_process(COMMAND python setup.py install --prefix=${CMAKE_INSTALL_PREFIX} WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/python)")
 
+IF (USE_PYTHON)
+	INSTALL(FILES ${CMAKE_BINARY_DIR}/python/setup.py 
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/singa/python
+		COMPONENT PySinga)
+	INSTALL(FILES ${CMAKE_BINARY_DIR}/python/singa/singa_wrap.py
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/singa/python/singa
+		COMPONENT PySinga)
+	INSTALL(FILES ${CMAKE_BINARY_DIR}/python/singa/_singa_wrap.so
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/singa/python/singa
+		COMPONENT PySinga)
+	INSTALL(DIRECTORY ${PROJECT_SOURCE_DIR}/python 
```

Comment:
Line 184 copies the Python **source** files `${PROJECT_SOURCE_DIR}/python `. The other four lines copy the Python **binary** files `${CMAKE_BINARY_DIR}/python`. So both of them are needed because this is a binary package not a source only package. 

If we copy only the sources, then the user must install the build tools and compile the Python module by himself.